Sources/StreamChat/APIClient/Endpoints/Payloads/UnknownEvent.swift

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+public protocol CustomEventPayload: Codable, Hashable {
+    static var eventType: String { get }
```

Comment:
we could make the `EventType` to be a string representable other than an `enum` so customers can declare their own types 🤔 But maybe it makes sense to keep `EventType` for system events only. WDYT?

Reply:
made the `EventType` string representable